### PR TITLE
Bello frontend

### PR DESCRIPTION
**Pull request by TunarshBee**

### CODE REVIEW

**Review by AyodeleJoshua — COMMENTED** [review]
please make a separate css file for ErrorPage and Error (e.g. error.css and errorPage.css) and import each css file in each component.

File: Web Application Codes/web/src/components/Error.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            <div>
+                <center>
+                    <div className='errorNav' style={navstyle}></div>
+
+                    <div className='errorBody' style={bodystyle} >
+                        <div className='topLogo' style={{ width: '40%', margin: 'auto', marginTop: '1%' }} >
+                            <svg className='Path_2076' viewBox='0 0.492 44.69 17.878' >
+                                <path
+                                    id='Path_2076'
+                                    d='M 0.6391686201095581 4.968416690826416 L 5.698694705963135 4.245887279510498 C 6.62128210067749 3.679137229919434 7.578700542449951 3.171795129776001 8.565767288208008 2.726577520370483
```
Comment (vheckthor) [inline Web Application Codes/web/src/components/Error.js:18]: what is all this long cryptic numbers? Are we running quantum algorithms?

**Review by vheckthor — CHANGES_REQUESTED** [review]
And also move the css as requested by Joshua

File: Web Application Codes/web/src/components/Error.js (hunk 1 above)
Comment (TunarshBee) [reply Web Application Codes/web/src/components/Error.js:18]: It's a svg tag for the logo

**Pull request closed without merging**